docs/en/install.md

```diff
@@ -96,9 +96,9 @@ These steps will install KubeVela controller and its dependency.
     helm install -n vela-system kubevela kubevela/vela-core --set useWebhook=false
```

Comment:
The above description about the default case also need to update , please describe the new mechanism a bit